### ISSUE TITLE
`sphinx-click` doesn't document click

### ISSUE THREAD

**Issue by t-sasatani** [Aharoni-Lab/mio#71]
The CLI documentation doesn't appear in `v0.5.0`, though `cli/main.rst` hasn't changed—just a note for returning later. 
- v0.5.0: https://miniscope-io.readthedocs.io/en/v0.5.0/cli/main.html
- v0.4.1: https://miniscope-io.readthedocs.io/en/v0.4.1/cli/main.html

**Comment by sneakers-the-rat** [comment]
Yeah I noticed that too, it builds locally but I think i messed smth up in the docs. Will add doc tests

**Comment by sneakers-the-rat** [comment]
oof wait i still broke this, reopening bc i need to convert .rst formatted click commands to myst

**Comment by sneakers-the-rat** [comment]
this works now!
